src/student/Main.java

```diff
@@ -0,0 +1,115 @@
+package student;
+
+import com.sun.scenario.effect.impl.sw.sse.SSEBlend_SRC_OUTPeer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        StudentManagement studentManagement = new StudentManagement();
+        ArrayList<Student> studentList = studentManagement.getList();
+        Scanner input = new Scanner(System.in);
+        int choose;
+        do {
+            showMenu();
+            System.out.println("Enter Choose Of You");
+            choose = Integer.parseInt(input.nextLine());
+            switch (choose){
+                case 1:
+                    Student std = getInfor();
+                    studentManagement.addStudent(std);
+                    break;
+                case 2:
+                    System.out.println("Enter The Student ID You Want To Delete");
+                    long inputID = Long.parseLong(input.nextLine());
+                    Student student = studentManagement.searchByID(inputID);
+                    if(student==null){
+                        break;
+                    }
+                    studentManagement.deleteStudent(student);
+                    System.out.println("Successful Implementation");
+                    break;
+                case 3:
+                    System.out.println("Enter The Student ID You Want To Search");
+                    int search = Integer.parseInt(input.nextLine());
+                    for(Student student1 : studentList){
+                        if(student1 == studentManagement.searchByID(search)){
+                            show(student1);
+                        }else{
+                            System.out.println("Not Found ID");
+                        }
+                    }
+                    break;
+                case 4:
+                    Collections.sort(studentList, new Comparator<Student>() {
+                        @Override
+                        public int compare(Student student, Student t1) {
+                            if(student.calcualteAverageScore()> t1.calcualteAverageScore()){
+                                return -1;
+                            }else if(student.calcualteAverageScore()<t1.calcualteAverageScore()){
+                                return 1;
+                            }else{
+                                return 0;
+                            }
+                        }
+                    });
+                    for(Student student1 : studentList){
+                        System.out.println("After Sort");
+                        show(student1);
+                    }
+                    break;
+                case 5:
+                    System.out.println("Enter Student Name You Want To Search");
+                    String inputName = input.nextLine();
+                    for(Student student2 : studentList){
```

Comment:
E ko cần for để duyệt lại ở đây nữa, e chỉ cần gọi hàm searchByName là ra Student e muốn tìm, nếu student == null thì thông báo ra màn hình

Reply:
dạ vâng